src/model/wallet.ts

```diff
@@ -287,12 +295,10 @@ export const walletInit = async (props: InitializeWalletParams): Promise<WalletI
 
 export const loadWallet = async <S extends Screen>({
   address,
-  showErrorIfNotLoaded = true,
```

Comment:
I didn't see any cases where we wouldn't want to show the error sheet when loading wallet, so removed this option.